featurebyte/models/observation_table.py

```diff
@@ -120,11 +120,14 @@ class ObservationTableModel(MaterializedTableModel):
         The input that defines how the observation table is created
     context_id: Optional[PydanticObjectId]
         The id of the context that the observation table is associated with
+    use_case_ids: Optional[List[PydanticObjectId]]
+        The ids of the use cases that the observation table is associated with
     """
 
     request_input: ObservationInput
     most_recent_point_in_time: StrictStr
     context_id: Optional[PydanticObjectId] = Field(default=None)
+    use_case_ids: Optional[List[PydanticObjectId]] = Field(default_factory=list)
```

Comment:
If `Field(default_factory=list)` is a list, we could simplify the type here as follows:
```
use_case_ids: List[PydanticObjectId]
```